components/script/script_thread.rs

```diff
@@ -2834,13 +2834,13 @@ impl ScriptThread {
                 }
             },
 
-            KeyEvent(ch, key, state, modifiers) => {
+            KeyboardEvent(key_event) => {
                 let document = match { self.documents.borrow().find_document(pipeline_id) } {
                     Some(document) => document,
                     None => return warn!("Message sent to closed pipeline {}.", pipeline_id),
                 };
-                document.dispatch_key_event(ch, key, state, modifiers);
-            },
+                document.dispatch_key_event(key_event);
+            }
```

Comment:
nit: I think `rustfmt` would ask to add the trailling comma back

Reply:
Thanks!